markdown_writer.py

```diff
@@ -180,7 +180,7 @@ def write_overall_metrics_table(
     columns,
     file,
 ):
-    """Write the overall metrics table to the markdown file."""
+    """Write the overall metrics tables to the markdown file."""
     file.write("| Metric | Average | Median | 90th percentile |\n")
```

Comment:
Maybe before doing any of this, one would have to check if the 1st stable will contain any data?

Something along the lines of this:

```
    if "Time to first response" in columns \
        or "Time to close" in columns \
        or "Time to answer" in columns \
        or labels:
````

Reply:
I did a quick test locally, and confirmed that this would indeed prevent the generation of the 1st table in this scenario.

I also confirmed that if at least one of these columns exists, the 1st table will be generated.

However we should probably add a real test case for this.

Reply:
I'm on it! Thanks @spier!

Reply:
Check out [3da633e](https://github.com/github/issue-metrics/pull/148/commits/3da633e12297baa497d452525ea5435a6cbab392)